release-1.7/release_team.md

```diff
@@ -1,12 +1,12 @@
 |  **Role** | **Name** (**GitHub/Slack ID**)  | **Shadow Name(s) (GitHub/Slack ID), ...**
 |  ------ | ------ | ------ |
-|  Lead | Dawn Chen (dchen1107) | Maru Newby (marun) |
-|  Secondary | | |
+|  Lead | Dawn Chen (dchen1107) | Shruti Ramesh (shrutir25) |
+|  Secondary |  | |
 |  Features |Ihor Dvoretskyi (idvoretskyi) | |
 |  Release branch |Wojciech Tyczynski (wojtek-t)| |
 |  Infra | | | |
 |  Docs |Andrew Chen (chenopis) | |
 |  Bugs | | | |
 |  Testing | | | |
-|  Manual upgrade | | | |
+|  Manual upgrade | Anthony Howe (anhowe) | | |
```

Comment:
manual + automatic upgrade can probably be combined at this point into one role served by Anthony